src/server/readpool/mod.rs

```diff
@@ -109,6 +109,18 @@ impl<T: futurepool::Context + 'static> ReadPool<T> {
         }
     }
 
+    /// TODO: Remove thus function. Only used to be compatible with endpoint.
+    #[inline]
+    pub fn get_max_tasks(&self) -> usize {
+        vec![
```

Comment:
Why create a vector?

Reply:
To calculate the max. Do you have better alternatives?

Reply:
Seems two if or two `cmp::max` can do the job perfectly.

Reply:
Yeah, exactly. I was just trying to prevent `cmp::max(cmp::max())` 😢 If you think it acceptable I can use this one.

Reply:
Why allocation is preferred? Not to mention that it's longer.

Reply:
Ok